### DISCUSSION TITLE
Mass delete mails which are held

### DISCUSSION THREAD

**Discussion by niels-przybilla** [postalserver/postal#1868] [Help with using Postal]
Hello Postal-Fans,

how can I delete mass delete a lot maisl which are in one specific mailserver ?

Many thanks for your help.

Best wishes

Niels

**Comment by CryptosWORLD** [comment]
is a way to mass manage queue and held messages ?
like retry delivery ,release message, cancel hold ..

**Reply by willpower232** [reply]
I'm afraid not, you'd have to open a lot of tabs to get the messages released if you want them to be.

**Reply by neskhan** [reply]
how do i delete held messages in postal?

**Comment by willpower232** [comment]
@niels-przybilla why do you need to delete the messages? Postal should automatically clean up the database after a certain period of time

**Reply by niels-przybilla** [reply]
because it was an old newsletter that should not be redelivered after 7 days. In this case it was for valentines day :-) And getting a newletter 7 days later would not be that good :-)

**Comment by CryptosWORLD** [comment]
personally i m in testing period , what i mean i have postal in live mode and send some mails in a small mail list that ALL emails are mine and all mails are ok 
i see a 1.1% bounce rate , a lot of mails in queued or held  and i wonder (as **new feature**) if there is more management related to (a Bolton that does the follows):

1. start all queued mails now
2. start specific domains queued  now
3. delete all queued mails now
4. delete specific domains queued  now
3. start all held mails now
4. start specific domains held now
5. delete all held mails now
6. delete specific domains held now
